cli/cmd/encore/gen.go

```diff
@@ -117,7 +117,7 @@ which may require the user-facing wrapper code to be manually generated.`,
 		Args: cobra.ExactArgs(0),
 		Run: func(cmd *cobra.Command, args []string) {
 			appRoot, _ := determineAppRoot()
-			ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+			ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
```

Comment:
I'd maybe go to 30 seconds. On shared CI machines, their can be really little CPU availability and I've seen before it can be bursty too, which means stuff which can complete sub-second on my machine can sometimes take 15 seconds on CI.